src/lib/cimb_translator/Common.cpp

```diff
@@ -19,27 +20,53 @@ using std::vector;
 namespace {
 	RGB getColor4(unsigned index)
 	{
-		// opencv uses BGR, but we don't have to conform to this tyranny
+		// opencv uses BGR, but we don't have to conform to its tyranny
 		static constexpr array<RGB, 4> colors = {
-		    RGB(0, 0xFF, 0xFF),
-		    RGB(0xFF, 0xFF, 0),
-		    RGB(0xFF, 0, 0xFF),
-		    RGB(0, 0xFF, 0)
+			RGB(0, 0xFF, 0),
+			RGB(0, 0xFF, 0xFF),
+			RGB(0xFF, 0xFF, 0),
+			RGB(0xFF, 0, 0xFF),
+		};
+		return colors[index];
+	}
+
+	RGB getColor4_old(unsigned index)
+	{
+		static constexpr array<RGB, 4> colors = {
+			RGB(0, 0xFF, 0xFF),
+			RGB(0xFF, 0xFF, 0),
+			RGB(0xFF, 0, 0xFF),
+			RGB(0, 0xFF, 0),
 		};
 		return colors[index];
 	}
 
 	RGB getColor8(unsigned index)
 	{
 		static constexpr array<RGB, 8> colors = {
-		    RGB(0, 0xFF, 0xFF), // cyan
-		    RGB(0x7F, 0x7F, 0xFF),  // mid-blue
-		    RGB(0xFF, 0, 0xFF), // magenta
-		    RGB(0xFF, 65, 65), // red
-		    RGB(0xFF, 0x9F, 0),  // orange
-		    RGB(0xFF, 0xFF, 0), // yellow
-		    RGB(0xFF, 0xFF, 0xFF),
-		    RGB(0, 0xFF, 0),
+			RGB(0, 0xFF, 0xFF), // cyan
```

Comment:
No 8-color configurations are supported by 0.6.0 -- but this is the correct (minimal bit errors) order of the existing colors.